HomePage/HomePage.de.json

```diff
@@ -27,13 +27,13 @@
   "RefreshSensors": "Sensorerkennung neu laden",
   "SaveConfig": "Konfiguration speichern",
   "SaveConfigAs": "Konfiguration speichern als ….",
-  "Settings": "Einstellungen",
+  "Settings": "Einst.",
   "Setup": "Einrichtung",
   "ShowHiddenCards": "Ausgeblendete Karten anzeigen",
   "Speeds": "Drehzahlen",
   "TemperatureUnitQuestion": "Um die Änderung der Temperatureinheit anzuwenden, wird die Konfiguration gespeichert und das Anwendungsfenster zurückgesetzt. Fortfahren?",
   "Theme": "Design",
```

Comment:
Use \r\n to force a newline between "Tray" and "icons",  like so:

Tray\r\nIcons